app/controllers/triannon/annotations_controller.rb

```diff
@@ -12,7 +13,11 @@ class AnnotationsController < ApplicationController
 
     # GET /annotations
     def index
-      redirect_to "/search"
+      if !params[:anno_root].blank?
+        redirect_to "/#{params[:anno_root]}/search"
+      else
+        redirect_to "/search"
```

Comment:
Can you use the url helpers provided by your routes here, e.g. `redirect_to search_url`?